src/platforms/javascript/common/install/cdn.mdx

```diff
@@ -1,114 +0,0 @@
----
-title: CDN
```

Comment:
It's not obvious to me that this CDN setup doc should be combined with the Loader setup doc. Would it be better if they remained separate?